modules/templates/global-elements.xml

```diff
@@ -198,7 +198,7 @@
                 <title>Date</title>
                 <default></default>
             </property>
-            <property id="dateFormat" type="text" variant="dateFormat">
+            <property id="dateFormat" type="text">
```

Comment:
Sorry this was correct - it was the default format which was incorrect, and the parsing of that in viewer.js